### PR TITLE
new files to Simon for PR

### PR DESCRIPTION
**Pull request by havegil**
New files added for the three pages per the design specs
## Description
The Acrobat SDK

New fIles added
## Motivation and Context
These are brand new files create for this project, as such there isn't an open issue as of yet.

## How Has This Been Tested?
This has been test locally and is currently under review at Proximitylab

Not a change

These are pure html/CSS files

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -9,7 +9,9 @@ GEM
       eventmachine (>= 0.12.9)
       http_parser.rb (~> 0.6.0)
     eventmachine (1.2.7)
+    eventmachine (1.2.7-x64-mingw32)
```
Comment (simonwex) [inline Gemfile.lock:12]: This and other windows-specific requirements shouldn't be in the gemfile lock. Please back these out.

File: _layouts/default.html [hunk 2]
```diff
@@ -1,23 +1,188 @@
 <!DOCTYPE html>
 <html class="spectrum--medium">
   <head>
-    <title>Adobe Developers - {{ page.title }}</title>
+    <title>SDK Developers Kit | {{ page.title }}</title>
```
Comment (simonwex) [inline _layouts/default.html:4]: SDK Developers kit shouldn't be in the default layout for the whole site. You should be updating page.title in the frontmatter of your pages.

File: _layouts/default.html [hunk 3]
```diff
@@ -1,23 +1,188 @@
...
+        var nPage = window.location.pathname;
+        nPage = nPage.substring(nPage.lastIndexOf('/') + 1);
+        var strNewClass = nPage.split('.')[0];
+        if(strNewClass == ""){strNewClass = "home"}
+        $('body').addClass(strNewClass);
+        console.log("You are in : "  + strNewClass)
+
+  </script>
+
+  <script src="js/prism.js"></script>
```
Comment (simonwex) [inline _layouts/default.html]: This feels like a lot of code to be including in the default layout. Also, we need comments here.

**Review by simonwex — CHANGES_REQUESTED** [review]
I'm getting the following error when I try to run this:

```
Liquid::SyntaxError: Liquid syntax error (line 97): Unknown tag 'prism'
  /Users/simon/.gem/ruby/2.5.0/gems/liquid-4.0.3/lib/liquid/document.rb:23:in `unknown_tag'
  /Users/simon/.gem/ruby/2.5.0/gems/liquid-4.0.3/lib/liquid/document.rb:11:in `block in parse'
```

**Comment by havegil** [comment]
Hi Simon,

Please run bundle install, it's in the Gem file.

Thanks, Gil

Kind Regards,
Gil

**Pull request merged**